server/manager/src/main/java/org/apache/accumulo/manager/tableOps/bulkVer2/CleanUpBulkImport.java

```diff
@@ -53,15 +52,15 @@ public CleanUpBulkImport(BulkInfo info) {
   public Repo<Manager> call(long tid, Manager manager) throws Exception {
     manager.updateBulkImportStatus(info.sourceDir, BulkImportState.CLEANUP);
     log.debug("removing the bulkDir processing flag file in " + info.bulkDir);
+    Ample metadataActions = manager.getContext().getAmple();
```

Comment:
could name the var `ample`, its shorter but I think its still readable.  This is a stylistic comment, please feel free to ignore it.

```suggestion
    Ample ample = manager.getContext().getAmple();
```